docs-content/index.html.md.erb

```diff
@@ -1,121 +1,109 @@
 ---
-title: YOUR-PRODUCT-NAME (Beta)
+title: Splunk OpenTelemetry Collector for VMware Tanzu (Beta)
 owner: Partners
 ---
 
 <strong><%= modified_date %></strong>
 
-**If this product is in beta, keep and edit the warning note below.**
 <p class="note warning">
 <strong>Warning: </strong>
-The PRODUCT-NAME for VMware Tanzu tile is currently in beta and is intended for evaluation and test purposes only.
+The Splunk OpenTelemetry Collector for VMware Tanzu tile is currently in beta and is intended for evaluation and test purposes only.
```

Comment:
It'd be nice if we could link to a definition of beta (I'm guessing Pivotal's).

Reply:
Just checked the word list they gave us in their instructions and we need to use experimental feature instead of beta. See https://docs.google.com/spreadsheets/d/1y-OI7LRZrVWRbs-ZplxFhNWRETAqFSSd9CpDPAu93W0/edit#gid=0.

Reply:
Just to clarify, here's the distinction:

```
Beta features are always with the intention of becoming a supported feature. We encourage the use of beta products. This is in contrast to an experimental feature, which we don't promise to support in the future.

Use experimental over beta feature when the feature can change or be removed depending on feedback.
```

So I guess the feature can change or be removed, but the intention is to become a supported feature. Also, the [product release is classified](https://docs.vmware.com/en/Tile-Developer-Guide/2.10/tile-dev-guide/releases.html) as `Public (Open) Beta`, which also added to my confusion around terminology here. I'll update to experimental feature.